models/Document/Editable/Relations.php

```diff
@@ -86,7 +86,7 @@ public function getDataEditmode(): array
             if ($element instanceof DataObject\Concrete) {
                 $return[] = [$element->getId(), $element->getRealFullPath(), DataObject::OBJECT_TYPE_OBJECT, $element->getClassName()];
             } elseif ($element instanceof DataObject\AbstractObject) {
-                $return[] = [$element->getId(), $element->getRealFullPath(), DataObject::OBJECT_TYPE_OBJECT, DataObject::OBJECT_TYPE_VARIANT, DataObject::OBJECT_TYPE_FOLDER];
+                $return[] = [$element->getId(), $element->getRealFullPath(), DataObject::OBJECT_TYPE_OBJECT, $element->getType()];
```

Comment:
Theoretically should be 
```suggestion
                $return[] = [$element->getId(), $element->getRealFullPath(), DataObject::OBJECT_TYPE_OBJECT, DataObject::OBJECT_TYPE_FOLDER];
```
to be consistent with
https://github.com/pimcore/admin-ui-classic-bundle/blob/1976b8f869ad450e1b3d2b9e4aebcc47f3487e7e/public/js/pimcore/document/editables/relations.js#L275